frontend/app/routing.js

```diff
@@ -79,7 +133,7 @@ angular.module('openproject')
       templateUrl: '/templates/work_packages.list.new.html'
     })
     .state('work-packages.list.details', {
-      url: '/{workPackageId:[0-9]+}?query_props',
+      url: '/details/{workPackageId:[0-9]+}?query_props',
```

Comment:
It's to differentiate between the fullscreen show view and the split view. If I don't change one of them both have `/work_packages/:id/:tab` as the URL. Changing the show view URL seemed more cumbersome to me.

Old links should still work, but they end up pointing to the fullscreen view instead of the split view now. IMO no big deal: Same information, different layout.